code/modules/mob/living/carbon/human/species_types/podpeople.dm

```diff
@@ -16,6 +16,7 @@
 	heatmod = 1.5
 	payday_modifier = 1.0
 	meat = /obj/item/food/meat/slab/human/mutant/plant
+	inert_mutation = /datum/mutation/harmonizing_pulses //BUBBER EDIT ADDITION - Unique Mutation
```

Comment:
Tiny nitpick, this line can be moved into `modular_zubbers/code/modules/mob/living/carbon/human/species_types/podpeople.dm`, looks good otherwise!